v2/manager.go

```diff
@@ -426,6 +432,7 @@ func (c *Manager) Stat() (*stats.Metrics, error) {
 		UsageLimit:            getStatFileContentUint64(filepath.Join(c.path, "memory.max")),
 		SwapUsage:             getStatFileContentUint64(filepath.Join(c.path, "memory.swap.current")),
 		SwapLimit:             getStatFileContentUint64(filepath.Join(c.path, "memory.swap.max")),
+		EventsMax:             getUint64Value("max", memoryEvents),
```

Comment:
Can we also add other fields?

Reply:
Do we need them? I mean, they used somewhere?

Reply:
Not now probably, bu

Reply:
Got it.Ok.